src/models/converters/transactions/tests/summary.rs

```diff
@@ -379,4 +379,41 @@ fn multisig_transaction_to_setttins_change_summary() {
     let actual = MultisigTransaction::to_transaction_summary(&multisig_tx, &mut mock_info_provider);
 
     assert_eq!(&expected, actual.unwrap().get(0).unwrap());
-}
\ No newline at end of file
+}
+
+#[test]
+fn multisig_transaction_to_custom_summary() {
+    let multisig_tx = serde_json::from_str::<MultisigTransaction>(crate::json::MULTISIG_TX_CUSTOM).unwrap();
+    let safe_info = serde_json::from_str::<SafeInfo>(crate::json::SAFE_WITH_MODULES).unwrap();
+    let token_info = serde_json::from_str::<TokenInfo>(crate::json::TOKEN_CRYPTO_KITTIES).unwrap();
+
+    let mut mock_info_provider = MockInfoProvider::new();
+    mock_info_provider
+        .expect_safe_info()
+        .times(1)
+        .return_once(move |_| Ok(safe_info));
+    mock_info_provider
+        .expect_token_info()
+        .times(1)
+        .return_once(move |_| Ok(token_info));
```

Comment:
Here it also seems a bit counter intuitive to be asking for the `token_info`

Reply:
Why? You can just return None

Reply:
True. I was wondering if it was a better option to simply not having to call that method at all.

Reply:
Just FYI, this is not an `Option` so I have to flat out return `Error` 😂